src/components/themed/FlipInput.js

```diff
@@ -69,7 +69,8 @@ export type FlipInputOwnProps = {
   headerText: string,
   headerLogo: string | void,
   headerCallback?: () => void,
-  keyboardVisible: boolean
+  keyboardVisible: boolean,
+  flipInputRef: any
```

Comment:
Isn't this of type `flipInputRef?: (current: FlipInput | undefined) => void`

Reply:
`flipInputRef: (FlipInput | null) => void` surprisingly works. Somehow I have an impression it would have an error because of how FlipInput is declared later on the code

Reply:
Then do `(element: any) => void` or such.